taskvine/test/vine_python_future_funcall.py

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env python
+
+import sys
+import ndcctools.taskvine as vine
+
+port_file = None
+try:
+    port_file = sys.argv[1]
+except IndexError:
+    sys.stderr.write("Usage: {} PORTFILE\n".format(sys.argv[0]))
+    raise
+
+
+# Define a library function
+def divide(dividend, divisor):
+    import math
+    return dividend / math.sqrt(divisor)
+
+
+# Create Executor
+executor = vine.Executor(port=9123, manager_name='vine_future_funcall_test', factory=False)
+print("listening on port {}".format(executor.port))
+with open(port_file, "w") as f:
+    f.write(str(executor.port))
+
+# Create library task
+print("creating library from functions...")
+libtask = executor.create_library_from_functions('test-library', divide, import_modules=None, add_env=False)
+
+# Install library on executor.manager
+executor.install_library(libtask)
+
+# Submit several tasks for execution
+print("creating future function call tasks...")
+t1 = executor.future_funcall('test-library', 'divide', 128, 4)
```

Comment:
We want the argument here to just be `t1` rather than `t1.future`. some handling might have to be done to make that work.